Tests/SonarScanner.MSBuild.Common.Test/ProcessRunnerTests.cs

```diff
@@ -57,13 +57,13 @@ public void ProcRunner_ExecutionFailed()
 
         var logger = new TestLogger();
         var args = new ProcessRunnerArguments(exeName, true);
-        using var runner = new ProcessRunner(logger);
+        var runner = new ProcessRunner(logger);
 
         // Act
         var success = runner.Execute(args);
```

Comment:
To avoid `success.Succeeded` below
```suggestion
        var result = runner.Execute(args);
```